client/makelunch.less

```diff
@@ -175,7 +175,18 @@ body.home .card {
       .btn { display: block; }
     }
   }
-  &:hover, &:focus {
+  @media (min-width: 768px) {
```

Comment:
Changed as requested.  We should really refactor the LESS into different files as it's pretty hard to find your way around at present.